gcn_classic_to_json/notices/GECAM_FLT/__init__.py

```diff
@@ -0,0 +1,39 @@
+from ... import utils
+
+
+def parse(bin):
+    bin[15]  # Unused. According to docs: '4 bytes for the future'
+    bin[27:39]  # Unused. According to docs: '32 bytes for the future'
+    bin[
+        11
+    ]  # Temporarily Unused. Should have a int->class_type mapping but incomplete documentation
```

Comment:
@jracusin, would you please help @athish-thiru find the integer to class name mapping for this field?